packages/manager/apps/web/client/app/hosting/email/activate/activate.routing.js

```diff
@@ -0,0 +1,30 @@
+import endsWith from 'lodash/endsWith';
+import filter from 'lodash/filter';
+import sortBy from 'lodash/sortBy';
+import union from 'lodash/union';
+
+import {
+  WEB_HOSTING_NOT_USABLE_DOMAINS,
+} from './activate.constants';
+
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('app.hosting.activate', {
+    url: '/activate',
+    component: 'webHostingEmailActivateComponent',
+    resolve: {
+      serviceName: /* @ngInject */ $transition$ => $transition$.params().productId,
+      user: /* @ngInject */ User => User.getUser(),
```

Comment:
`user` is already resolved at the app root state 
https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/app.routes.js#L11